src/Microsoft.DotNet.CodeFormatter.Analyzers/OptimizeNamespaceImportsOptions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+
+using Microsoft.CodeAnalysis.Options;
+ 
+namespace Microsoft.DotNet.CodeFormatter.Analyzers
+{
+    [Export(typeof(IOptionsProvider)), Shared]
+    internal class OptimizeNamespaceImportsOptions : IOptionsProvider
```

Comment:
Is there any coupling between the options that an analyzer declares that it can understand (the ones here) and the analyzer itself? If an analyzer doesnt export an optionprovider but calls PropertyBag.TryGetProperty("someString") and someString was specified in the file, it looks like it would just work? If so, that would be a good thing right? That way the OptionsProvider is an optional declaration to give default values but can be skipped altogether - is that the design?